clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/services/ContainerInformationService.java

```diff
@@ -86,19 +85,12 @@ public ContainerInformationService(
 
       healthMetrics.add(loadBalancerHealth);
     } else {
-      List<LoadBalancer> loadBalancers = service.getLoadBalancers();
-      // There should only be 1 based on AWS documentation.
-      if (loadBalancers.size() == 1) {
-        Map<String, Object> loadBalancerHealth = new HashMap<>();
-        loadBalancerHealth.put("instanceId", taskId);
-        loadBalancerHealth.put("state", taskHealth.getState());
-        loadBalancerHealth.put("type", taskHealth.getType());
-
-        healthMetrics.add(loadBalancerHealth);
-      } else if (loadBalancers.size() >= 2) {
-        throw new IllegalArgumentException(
-            "Cannot have more than 1 load balancer while checking ECS health.");
-      }
+      Map<String, Object> loadBalancerHealth = new HashMap<>();
+      loadBalancerHealth.put("instanceId", taskId);
+      loadBalancerHealth.put("state", taskHealth.getState());
+      loadBalancerHealth.put("type", taskHealth.getType());
+
+      healthMetrics.add(loadBalancerHealth);
```

Comment:
FYI I think this data is used by deck in the infrastructure view to list LB info - we should make sure this is still displaying correctly when multiple load balancers are present.

Reply:
It currently only displays one LB, is that something we can fix in a separate PR (don't want to put too much into this 1 pr)?